scripts/ci/run_perftest.sh

```diff
@@ -342,16 +333,16 @@ for usecase in iot metaquery multi-measurement; do
         $working_dir/test-query-$format-$query_usecase-$type.json
 
       # Restart daemon between query tests.
-      systemctl stop influxdb
-      systemctl unmask influxdb.service
-      systemctl start influxdb
+      systemctl restart influxdb
   done
 
   # Delete DB to start anew.
   curl -X DELETE -H "Authorization: Token ${TEST_TOKEN}" http://${NGINX_HOST}:8086/api/v2/buckets/$(bucket_id)
+  rm -rf "$USECASE_DIR"
 done
 
-echo "Using Telegraph to report results from the following files:"
-ls $working_dir
-
-telegraf --debug --once
+if [ "${TEST_RECORD_RESULTS}" = "true" ] ; then
```

Comment:
Another thought: If the results aren't being recorded, instead of totally omitting this step it would be nice to instead run it with the `--test` flag: https://github.com/influxdata/telegraf#run-a-single-telegraf-collection-outputting-metrics-to-stdout so that we can see what would be sent to the cloud dashboard in the logs.

I usually just let it report the results anyway on testing branches, since the data on the dashboards is filtered by branch. Having the test output in the logs might make it even easier though, that way you wouldn't have to go back into the cloud dashboards and clean up the extra data.